search-app/src/cmr/search/results_handlers/stac_results_handler.clj

```diff
@@ -218,3 +223,63 @@
 (defmethod qs/single-result->response [:granule :stac]
   [context query results]
   (single-result->response context query results))
+
+(defn- get-fc-links
+  "Returns the links for feature collection"
+  [context query hits]
+  (let [{:keys [page-size offset]} query
+        page-num (if offset
+                   (inc (quot offset page-size))
+                   1)
+        item-count (if offset
+                     (+ offset page-size)
+                     page-size)
+        allowed-hits (min hits MAX_RESULT_WINDOW)
+        match-1 (str "page_num=" page-num)
+        match-2 (str "page-num=" page-num)
```

Comment:
Why have match-2?  it is the same as match-1 and I don't see it changing.  for page-num-pattern it looks the same if you used match-1 match-1 match-1 match-1.

Reply:
It is matching `page_num` and `page-num` both and in beginning of the query string or end of the query string.